orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -3131,3 +3142,36 @@ def test_debug_options_log_level_validation_fails_on_type_mismatch():
     with pytest.raises(Exception) as ex_info:
         _ = DebugOptions(log_level=log_level)
     assert f"Expected log_level of type LogLevel, got {type(log_level)}." in str(ex_info.value)
+
+def test_duplicated_output_eval():
+    device = 'cuda'
+    N, D_in, H, D_out = 64, 784, 500, 10
+    pt_model = DuplicatedOutputNet(D_in, D_out).to(device)
+    pt_model.eval()
+    ort_model = ORTModule(copy.deepcopy(pt_model))
+    ort_model.eval()
+    x = torch.randn(N, D_in, device=device)
+    y = copy.deepcopy(x)
+
+    pt_out = pt_model(x)
+    ort_out = ort_model(y)
+
+    _test_helpers.assert_values_are_close(ort_out[0], pt_out[0])
+    _test_helpers.assert_values_are_close(ort_out[1]['inner_states'][0], pt_out[1]['inner_states'][0])
+    _test_helpers.assert_values_are_close(ort_out[1]['inner_states'][1], pt_out[1]['inner_states'][1])
+
+def test_duplicated_output_no_grad():
```

Comment:
Can merge these two tests with pytest.mark.parametrize ? 
Move DuplicatedOutputNet into function's scope if only one test function is needed.

Reply:
combined the similar tests into a single test.